shared-libs/contacts/test/unit/places.spec.js

```diff
@@ -474,7 +516,7 @@ describe('places controller', () => {
       db.medic.post.callsFake(doc => {
         chai.expect(doc.parent._id).to.equal('a');
         chai.expect(doc.parent.name).to.equal(undefined); // minified
-        return Promise.resolve({id: 'x', rev: 'y'});
+        return Promise.resolve({ id: 'x', rev: 'y' });
       });
       return controller.updatePlace('123', data).then(actual => {
         chai.expect(actual).to.deep.equal({ id: 'x', rev: 'y' });
```

Comment:
Can you please add some unit tests where:
- the contact is not valid
- we create a person that is not a `person` type (configurable hierarchies with multiple person types)

Reply:
I've added 2 tests that should trigger errors in contact validation, is it worth adding more considering we already test for this elsewhere?